web/profiles/custom/yalesites_profile/modules/custom/ys_alert/ys_alert.module

```diff
@@ -24,3 +24,17 @@ function ys_alert_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function ys_alert_preprocess_page(&$variables) {
+  $config = \Drupal::config('ys_alert.settings');
+  $variables['alert'] = $config->get('alert');
```

Comment:
I don't think you need this line to set the `$variables['alert']` - that was in the `ys_themes` module to set the global theme for twig templates, but for the alert, you should only need the `$config` to send to the Drupal service. Otherwise, tested this out and all works great!

Reply:
You are correct.  Thanks for catching that.  The config is all we needed.  Thanks again for your help!